commit b3b031b07cc5909aaf964f9d4cf46f6097769320
Author: Steve Dickson <steved@redhat.com>
Date:   Wed Aug 21 14:40:22 2013 -0400

    rpcbind: rpcuser not being set in Makefile.am
    
    Commit 8d7a0708 cause a regression where the rpcuser id was not
    being set, which in turn cause rpcbind to immediately exit.
    This patch removes the extra ',' that was in the AC_ARG_WITH
    statement in the configure.ac file.
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/configure.ac b/configure.ac
index 2b67720..1cf42d3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -23,7 +23,7 @@ AC_ARG_WITH([statedir],
 AC_SUBST([statedir], [$with_statedir])
 
 AC_ARG_WITH([rpcuser],
-  AS_HELP_STRING([--with-rpcuser=ARG], [use ARG for RPC @<:@default=root@:>@]),
+  AS_HELP_STRING([--with-rpcuser=ARG], [use ARG for RPC @<:@default=root@:>@])
   ,, [with_rpcuser=root])
 AC_SUBST([rpcuser], [$with_rpcuser])
  
